src/pool/srv_pool.c

```diff
@@ -7404,7 +7405,7 @@ pool_update_map_internal(uuid_t pool_uuid, unsigned int opc, bool exclude_rank,
 
 	rc = pool_svc_update_map_internal(svc, opc, exclude_rank, NULL, 0, NULL, tgts, tgt_addrs,
 					  hint, p_updated, map_version_p, tgt_map_ver,
-					  inval_tgt_addrs, MUS_DMG, true);
+					  inval_tgt_addrs, MUS_DMG, 0);
```

Comment:
why does this not specify flags=POOL_TGT_UPDATE_SKIP_RF_CHECK in the final argument (to agree with skip_rf_check=true in the original code?

I realize there were 3 uses of pool_update_map_internal() previously, and one of them (related to no_data_sync) was removed in this patch - is that the only usage that required skip RF check?